controllers/client.rb

```diff
@@ -143,6 +141,14 @@ def request(path, payload = nil, server_sent_events: nil, request_method: 'POST'
         raise Errors::RequestError.new(e.message, request: e, payload:)
       end
 
+      def faraday
+        Faraday.new(request: @request_options) do |faraday|
+          faraday.adapter *@faraday_adapter
```

Comment:
This is the line I needed to modify. The splat is required so I can pass in `[:test, stubs]` as the adapter.